zaza/openstack/charm_tests/aodh/setup.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Code for setting up aodh tests."""
+
+import zaza.openstack.configure.telemetry
+
+
+def ceilometer_upgrade():
```

Comment:
Why not just reference the ``....ceilometer.setup....`` configure function from the charms tests.yaml?

Reply:
Ah, good catch, I need to rebase, ceilometer in zaza is a recent addition